developer_guide/akeneo_in_a_nutshell/index.rst

```diff
@@ -0,0 +1,79 @@
+Akeneo PIM in a nutshell
+========================
+
+What is a PIM?
+**************
+
+A Product Information Management (PIM) solution is aimed to centralize all the marketing data, to enrich, translate and prepare it for exports to multiple channels.
+It is a productivity tool helping the contributors to serve the product information in different languages and for different purposes.
+
+Akeneo is the first open source PIM made for non-technical users.
+Easy to use and very flexible, the solution can adapt to your product organization and processes, not the other way around.
+Available in two editions, Community, and Enterprise, you can choose the one that fits your needs.
+
+What is a product?
+------------------
+
+Akeneo provides a powerful and very flexible way to structure your products using an Entity Attribute Value (i.e. Product - Attribute - ProductValue) model.
```

Comment:
Ok, right now I think the difference between MySQL and hybrid storage shouldn't be part of an introduction but be explained somewhere (if it's not done right now).

I will complete this part